SU-SWS/stanford_basic
# Fix font awesome icons for far and fas sets

# READY FOR REVIEW

# Summary
- Fix FA icons on `<p>` tags for `far` and `fas` sets

# Need Review By (Date)
- February 2022

# Urgency
- medium

# Steps to Test
1. checkout this branch
2. clear browser caches
3. Use the following html and verify the icons display
```
<p class="fab fa-adn">FAB</p>

<p class="fas fa-address-book">FAS</p>

<p class="far fa-address-book">FAR</p>

<div class="fab fa-adn">FAB</div>

<div class="fas fa-address-book">FAS</div>

<div class="far fa-address-book">FAR</div>
```

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
